gpMgmt/bin/gppylib/gparray.py

```diff
@@ -959,6 +959,9 @@ def initFromCatalog(dbURL, utility=False):
             if not version.isVersionCurrentRelease():
                 raise Exception("Cannot connect to GPDB version %s from installed version %s"%(version.getVersionRelease(), MAIN_VERSION[0]))
 
+            # Get gp_internal_is_singlenode GUC
+            is_singlenode = dbconn.querySingleton(conn, "SHOW gp_internal_is_singlenode") == "on"
```

Comment:
what about config to true ?

Reply:
the variable `is_singlenode` is used in `gpstart`. when set to true, after `_startCoordinator()` and `_start_standby()` in the Phase 1 (start everything in utility mode) the script exited immediately:

<img width="773" alt="image" src="https://github.com/cloudberrydb/cloudberrydb/assets/7938763/0d018ca5-8bd5-4bfd-85cf-2761a8517b9e">

more on phase 1 & 2 of gpstart: https://github.com/orgs/cloudberrydb/discussions/188

Reply:
but true is equal to on .

Reply:
you mean `SHOW gp_internal_is_singlenode` could returns `true` instead of ` on`?